recipe/meta.yaml

```diff
@@ -19,6 +21,7 @@ requirements:
     # perl is required to run the tests on UNIX.
     - perl  # [unix]
     - pkg-config  # [unix]
+    - patch     # [s390x or ppc64le]
```

Comment:
See previous comment about the patch name. I would just make this selector `[unix]` and not worry if it isn't actually used.

Reply:
Selectors were removed and still built